cpp/ql/lib/semmle/code/cpp/ir/implementation/raw/internal/TranslatedFunction.qll

```diff
@@ -857,7 +857,12 @@ class TranslatedParameterReadEffect extends TranslatedReadEffect, TTranslatedPar
   }
 
   final override IRVariable getInstructionVariable(InstructionTag tag) {
-    tag = OnlyInstructionTag() and
+    (
+      tag = OnlyInstructionTag() or
+      tag = InitializerStoreTag() or
+      tag = InitializerVariableAddressTag() or
+      tag = InitializerIndirectStoreTag()
+    ) and
```

Comment:
Apologies for not being clear enough. This is a problem on the extractor side. This should not be fixed here.